app/models/auth_user.rb

```diff
@@ -14,6 +14,14 @@ def jwt_subject
     auth0_id_string
   end
 
+  def generate_token
+    request_env = {}
+    Warden::JWTAuth::Hooks.new.send(:add_token_to_env,
```

Comment:
can't we put this on a general place? WardenHelper or w/e, so we can use that one?

Reply:
Yes, but also no. I think it's too tied to this class.

You would have to move the jwt_subject function as well, but that is too closely tied to the model.  We're generating a token for an auth_user here, and we're not generating a token (we can't generate a token) for any other model in the database. And we're not duplicating this code anywhere. So I think this belongs in auth_user.